saleor/graphql/payment/mutations/transaction/transaction_process.py

```diff
@@ -155,7 +157,7 @@ def perform_mutation(cls, root, info, *, id, data=None):
             info, id, only_type="TransactionItem", field="token"
         )
         transaction_item = cast(payment_models.TransactionItem, transaction_item)
-        events = transaction_item.events.all()
+        events = transaction_item.events.filter()
```

Comment:
Is this on purpose? `filter` and `all` will work the same, or maybe some filtering is missing here?

Reply:
no it is not, fixing